tests/components/totalconnect/test_alarm_control_panel.py

```diff
@@ -197,3 +212,135 @@ async def test_disarm_invalid_usercode(hass):
             await hass.async_block_till_done()
         assert f"{err.value}" == "TotalConnect failed to disarm test."
         assert hass.states.get(ENTITY_ID).state == STATE_ALARM_ARMED_AWAY
+
+
+async def test_arm_night_success(hass):
+    """Test arm night method success."""
+    responses = [RESPONSE_DISARMED, RESPONSE_ARM_SUCCESS, RESPONSE_ARMED_NIGHT]
+    with patch(
+        "homeassistant.components.totalconnect.TotalConnectClient.TotalConnectClient.request",
+        side_effect=responses,
+    ):
+        await setup_platform(hass, ALARM_DOMAIN)
+        assert STATE_ALARM_DISARMED == hass.states.get(ENTITY_ID).state
```

Comment:
Please end with the expected value in assertions.

Reply:
changed